server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+var cors = require('cors')
+const app = express();
+
+app.use(cors())
+app.use(bodyParser.urlencoded({ extended: true }))
+
+app.use(bodyParser.json())
+
+app.get('/', (req, res) => {
+    res.json({ "message": "Welcome" });
+});
+
+app.get('/products', (req, res) => {
+    const limit = req.query.limit;
+    const page = req.query.page;
+    const offset = page * limit;
+    console.log(limit, page);
+    const obj = require("./src/data.json");
+    res.json(obj.splice(offset+1, offset + limit));
```

Comment:
The above logic will not work as expected,
a suggested possible approach 1 is as follows:
```
app.get('/products', (req, res) => {
    const limit = req.query.limit;
    const page = req.query.page;
    const offset = page > 0 ? (page * limit) : 0;
    //The follwing lines mock the DB Query behaviour START
    let products = Object.assign([], productsMockData);    
    let pagingatedData = products.splice(offset , limit);
    //Equivalent Query - `select * from products limit ${limit} offset ${offset}`
    //The follwing lines mock the DB Query behaviour END
    res.json(pagingatedData);
});
```